src/seedsigner/gui/screens/seed_screens.py

```diff
@@ -1695,3 +1695,211 @@ def __post_init__(self):
             screen_y=derivation_path_display.screen_y + derivation_path_display.height + 2*GUIConstants.COMPONENT_PADDING,
         )
         self.components.append(address_display)
+
+
+
+@dataclass
+class NumericEntryScreen(BaseTopNavScreen):
+    title: str = ""
+    entered_number: str = ""
+
+    def __post_init__(self):
+        super().__post_init__()
+
+        keys_number = "0123456789"
```

Comment:
For the purposes of entering the "SLIP-39 Threshold" in `SeedEntryShamirThresholdView`, it doesn't make sense to offer `0` on this keyboard. I assume that `1` is also nonsensical?